### ISSUE TITLE
Suport for fieldDelimiter in the Bigquery table.load_job

### ISSUE THREAD

**Issue by gramos74** [GoogleCloudPlatform/gcloud-ruby#370]
Right now there is no a option to set a delimiter for the file to be uploaded, so only csv files can be uploaded.

Our processes generate tsv files ( tab delimiter ) and would be great to upload those files directly because the tab delimiter is already supported by Bigquery. [ https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.load ]

Could be implemented this option for the next release ? 

I could try to do it myself and create a pull request...

**Comment by quartzmo** [comment]
@gramos74 You are right, this option is missing from Table#load. As far as I can tell, it can and should be added.

Adding it should be fairly simple if you are interested in contributing the change, and I would be happy to help you do it today if you have time. Have you read our [CONTRIBUTING](CONTRIBUTING.md) guide?

**Comment by gramos74** [comment]
Hi
Yes, Will be great for me do it. 

Yesterday I changed the Connection file locally and I tested the upload and worked fine. The thing is that I had problems running the tests.
Let me try again and I will reach you later.

**Comment by quartzmo** [comment]
Great! You read about the [contributor license agreement](https://github.com/GoogleCloudPlatform/gcloud-ruby/blob/master/CONTRIBUTING.md#contributor-license-agreements), right?

If you need to open a separate issue to get help with running tests, go ahead.

**Comment by gramos74** [comment]
Hi,

Pull request created.

**Comment by quartzmo** [comment]
Fixed by PR #371, thanks @gramos74.
